### ISSUE TITLE
Mac desktop app icon always grayed out - even when the app is in focus

### ISSUE THREAD

**Issue by VeljkoTN** [toggl/toggldesktop#2543]
A user on the 7.4.235 version of the app shared the below screenshot showing that the top menu bar is not giving him the Toggl account menu and the Toggl icon always being gray even when the app is in focus.

![image](https://user-images.githubusercontent.com/36152180/44389256-a62d5b80-a52a-11e8-997c-90fb1e1525ee.png)

The user us on Mac OS 10.13.6

**Comment by NghiaTranUIT** [comment]
It seems that I'm able to reproduce this issue by fortune 👨‍💻

Basically, when `Show Dock Icon` is `OFF`, and click on `Show` button on Menu, we are unable to access the Toggl Menu when focusing in the app. (see the GIF)

![2018-11-19 20 55 34](https://user-images.githubusercontent.com/5878421/48711433-dd043200-ec3d-11e8-9cef-c8e2cffd12f7.gif)

**Comment by IndrekV** [comment]
I've looked into this issue and it seems that not showing the menu is quite common with applications that do not have dock icon. Usually they are apps that only have menubar icon and all actions can be triggered from the menubar icon menu. I'm not 100% sure how Apple's Human Interface Guide addresses it as I did not find clear description about expected behavior for apps that do not have Dock icon.

In this sense the behavior of showing menu only when Dock icon is visible is correct.
